src/translator/response_builder.h

```diff
@@ -86,6 +91,8 @@ class ResponseBuilder {
   std::promise<Response> promise_; //  To be set when callback triggered and
                                    //  after Response constructed.
   AnnotatedText source_;
+
+  AlignedMemory *qualityEstimator_;
```

Comment:
`const QualityEstimator &qualityEstimator`. Consume `AlignedMemory` to instantiate `QualityEstimator` somewhere in `Service`.